source/features/remove-projects-tab.tsx

```diff
@@ -1,25 +1,22 @@
 import React from 'dom-chef';
 import select from 'select-dom';
-import onetime from 'onetime';
 import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '../libs/features';
 
-const addNewProjectLink = onetime(() => {
-	if (pageDetect.isUserProfile()) {
-		// The link already exists on our profile,
-		// and we can't create projects on others' profiles
-		return;
-	}
+function getProjectsTab() {
+	return elementReady([
+		'[data-hotkey="g b"]:not(.selected)', // In organizations and repos
```

Comment:
I like this, but this also means that if you're on the Projects tab, `addNewProjectLink` also won't run

Reply:
Done